scripts/checkMissingTranslations.ts

```diff
@@ -33,11 +33,11 @@ const checkRecursiveTranslation = (
   defaultKeys.forEach((key) => {
     if (!comparingLanguageObject[key]) {
       console.warn(
-        warning(
-          `The key ${key} is not present for path ${searchingPath.join(
-            '-->',
-          )} and language ${comparingLanguage}`,
-        ),
+        // warning(
+        `The key ${key} is not present for path ${searchingPath.join(
+          '-->',
+        )} and language ${comparingLanguage}`,
+        // ),
       )
```

Comment:
Unfortunately i am completely reworking this script. It will have conflicts for sure. Lets wait my commit and then act afterwards.

Reply:
I couldn't start project with 'npm run start' command because of chalk library import not working. 

The only intent was to be able to run project.

Reply:
You don't have to do it anymore. I fixed it on master :)